draft-ietf-rats-eat.md

```diff
@@ -2039,7 +2039,7 @@ This early allocation will presumably complete correctly
 * Claim Name: SW Evidence
 * Claim Description: Measurements of the SW, memory configuration and such on the entity
 * JWT Claim Name: "swevidence"
-* Claim Key: TBD274
+* Claim Key: TBD
```

Comment:
Why should a Verifier be allowed to include this Claim in Attestation Results. The descriptive text clearly implies that this Claim must originate from an Attester. What is the purpose of allowing this for Attestation Results?